src/interfaces/steps-interface.ts

```diff
@@ -4,5 +4,6 @@ export interface StepsIF {
   icon: string
   text: string
   to: string,
+  disabled: boolean,
```

Comment:
Is this property always used? If not then change to `disabled?: boolean`.

Reply:
I recall getting a TS error for this being optional but I'll take a look again.

Reply:
Looks good.